packages/devtools_app/integration_test/test/live_connection/app_test.dart

```diff
@@ -25,14 +25,18 @@ void main() {
     await resetHistory();
   });
 
-  testWidgets('connect to app and switch tabs', (tester) async {
-    await pumpAndConnectDevTools(tester, testApp);
-
-    // For the sake of this test, do not show extension screens by default.
-    preferences.devToolsExtensions.showOnlyEnabledExtensions.value = true;
-    await tester.pumpAndSettle(shortPumpDuration);
-
-    logStatus('verify that we can load each DevTools screen');
-    await navigateThroughDevToolsScreens(tester, connectedToApp: true);
-  });
+  testWidgets(
+    'connect to app and switch tabs',
+    timeout: const Timeout(Duration(minutes: 2)),
```

Comment:
just for consistency, can we add consts `shortTimeout`, `mediumTimeout` and `longTimeout` and put them in integration_test/test_infra/_utils.dart?